sql_scripts/5_styles.sql

```diff
@@ -0,0 +1,106 @@
+CREATE TABLE styles
```

Comment:
Make exucution stateful by adding IF NOT EXISTS, cause you will receive runtime error during second deployment. Also declare schema